utils/post_utils.jsx

```diff
@@ -53,6 +53,28 @@ export function getImageSrc(src, hasImageProxy) {
     return src;
 }
 
+export function changeToMp4Src(src, hasImageProxy) {
```

Comment:
Shouldn't these links be changed to mp4 and jpg before being passed to the image proxy?

Reply:
I made the suggested changes! Let me know if it is ok :)